### PR TITLE
api changes

### PR DESCRIPTION
**Pull request by mirnawong1**
adding conceptual and use cases on the initial [metadata api page](https://docs.getdbt.com/docs/dbt-cloud-apis/metadata-api)

## What are you changing in this pull request and why?
Making this change per PM request

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by runleonarun** [comment]
@mirnawong1 I am working on a [PR](https://github.com/dbt-labs/docs.getdbt.com/issues/2785) that touches this page. I'm updating references to the Access URL.

**Comment by mirnawong1** [comment]
thanks @runleonarun - should i branch off your fix-access-URLs branch to avoid merge conflicts?

File: website/docs/docs/dbt-cloud-apis/metadata-api.md [hunk 1]
```diff
@@ -3,13 +3,39 @@ title: "Metadata API"
...
+Metadata is data that provides information about other data and it's generated every time dbt Cloud runs a project. The metadata contains information on the accuracy, recency, configuration, and structure of the <Term id="view">views</Term> and tables in the warehouse.
+
+<!-- reworded original: Every time that dbt Cloud runs a dbt project, it generates metadata which pertains to the accuracy, recency, configuration, and structure of the views and tables in the warehouse. -->
+
+ dbt Cloud serves a GraphQL API which supports arbitrary queries over this metadata. This API is an incredibly rich resource for evaluating data health long-term or at a moment-in-time. 
+ 
+ The endpoint for this API is `https://metadata.cloud.getdbt.com/graphql`.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-api.md]: @runleonarun making changes to this section

**Comment by mirnawong1** [comment]
adding @runleonarun 's api access url text to this pr per meeting

File: website/docs/docs/dbt-cloud-apis/metadata-api.md [hunk 2]
```diff
@@ -3,13 +3,35 @@ title: "Metadata API"
...
 ---
 
-## About the metadata API
+The dbt Cloud Metadata API helps organizations analyze and improve their data. You can use the API to:
+ - Power downstream integrations so users discover and understand data for analysis
+ - Ensure data quality 
+ - Increase the efficiency of dbt operations
 
-Every time that dbt Cloud runs a dbt project, it generates metadata which pertains to the accuracy, recency, configuration, and structure of the <Term id="view">views</Term> and tables in the warehouse. dbt Cloud serves a GraphQL API which supports arbitrary queries over this metadata; the endpoint for this API is `https://metadata.cloud.getdbt.com/graphql`. This API is an incredibly rich resource for evaluating data health longitudinally or at a point in time.
+Metadata is data that provides information about other data and it's generated every time dbt Cloud runs a project. The metadata contains information on the accuracy, recency, configuration, and structure of the <Term id="view">views</Term> and tables in the warehouse.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/metadata-api.md]:
```suggestion
Metadata provides information about other data.  Every time dbt Cloud runs a project, it generates metadata that contains information on the accuracy, recency, configuration, and structure of the <Term id="view">views</Term> and tables in the warehouse.
```

File: website/docs/docs/dbt-cloud-apis/metadata-api.md [hunk 3]
```diff
@@ -3,13 +3,35 @@ title: "Metadata API"
...
+
+<!-- reworded original: Every time that dbt Cloud runs a dbt project, it generates metadata which pertains to the accuracy, recency, configuration, and structure of the views and tables in the warehouse. -->
+
+ dbt Cloud serves a [GraphQL API](https://metadata.cloud.getdbt.com/graphql) which supports arbitrary queries over this metadata. This API is an incredibly rich resource for evaluating data health long-term or at a moment-in-time.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/metadata-api.md]: Removing a stray space. 
```suggestion
dbt Cloud serves a [GraphQL API](https://metadata.cloud.getdbt.com/graphql) which supports arbitrary queries over this metadata. This API is an incredibly rich resource for evaluating data health long-term or at a moment-in-time. 
```

File: website/docs/docs/dbt-cloud-apis/metadata-api.md [hunk 4]
```diff
@@ -3,13 +3,35 @@ title: "Metadata API"
...
+ 
+ The endpoint for this API is `https://metadata.YOUR_ACCESS_URL/graphql`. Replace `YOUR_ACCESS_URL` with the appropriate [Access URL](/docs/deploy/regions-ip-addresses) for your region and plan. For example, if your region is North America multi tenant, your endpoint is `https://metadata.cloud.getdbt.com/graphql`. 
 
 ## Prerequisites
 
-The metadata API is available to accounts on the _Team_ and _Enterprise_ plans, for any version >= dbt v0.19.0. Please note that artifacts generated with a version of dbt _less than_ v0.19.0 will not be accessible via the dbt Cloud metadata API. For information on upgrading, see "[Version migration guides](https://docs.getdbt.com/guides/migration/versions)."
+The Metadata API is available to:
+- [Multi tenant](/docs/deploy/regions-ip-addresses) accounts on the [Team or Enterprise plans](https://www.getdbt.com/pricing/), 
+- Projects on dbt version v0.19.0 or higher. 
+    * Artifacts generated on dbt version lower tha v0.19.0 will not be accessible via the Metadata API. Refer to [Version migration guides](https://docs.getdbt.com/guides/migration/versions) to upgrade.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/metadata-api.md]:
```suggestion
    * The Metadata API cannot access artifacts generated on dbt version lower than v0.19.0. Refer to [Version migration guides](https://docs.getdbt.com/guides/migration/versions) to upgrade. 
```

**Review by runleonarun — COMMENTED** [review]
Looks good! 🚢 it! I will add these few comments and when Cameron gives the 👍🏻 we can merge this.

**Comment by cafzal** [comment]
Thanks @mirnawong1 ! Good to 🚢 . Looking forward to collaborating more on the use case guides soon

**Review by runleonarun — APPROVED** [review]
LGTM

**Pull request merged**